tsconfig.json

```diff
@@ -26,6 +26,6 @@
     "verbatimModuleSyntax": true,
     "typeRoots": ["node_modules/@types"]
   },
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "mock/**/*"],
```

Comment:
Can we move `mock` directory to `src/` directory? Or would this affect the paths in imports? Let's avoid `@expensify/react-native-live-markdown/src/mock` if possible (with `src/mock`)

Reply:
It will affect the import path indeed. It's better to keep the path as `@expensify/react-native-live-markdown/mock`